server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
 
 === Authentication Flows
 
-An _authentication flow_ is a container for all authentications, screens, and actions that must happen during login, registration, and other
-{project_name} workflows.
-If you go to the admin console `Authentication` left menu item and go to the `Flows` tab, you can view all the defined flows
-in the system and what actions and checks each flow requires.
+An _authentication flow_ is a container of authentications, screens, and actions during login, registration, and other {project_name} workflows.
+To view all the flows, actions and checks each flow requires:
+
+.Procedure
+. Click *Authentication* in the menu.
+. Click the *Flows* tab.
 
 ==== Built-in flows
 
-{project_name} comes with a certain number of built-in flows. These flows cannot be modified, but the requirements can be modified to
-suit your needs.
+{project_name} has several built-in flows. It is not possible to modify these flows, but it is possible to alter the flow's requirements to suit your needs.
 
-This section does a walk-through of the built-in browser login flow.  In the
-left drop-down list select `browser` to come to the screen shown below:
+In the drop-down list, select `browser` to display the Browser Flow screen.
 
 .Browser Flow
```

Comment:
repeat title after filename
[browser flow]

Reply:
Not sure what this means.

Reply:
Sorry it was not clear.   I mean that for the visually impaired the title (.Browser flow) should be also be on the next line in brackets afterwards. Example in our guidelines here:

https://docs.google.com/document/d/1H-oPMTTBKAJ8UuQ6uBavGIy-CPXV75SYNGeAK9WmpYY/edit#heading=h.kb3931fh96y4